drake/examples/TwoWheeledInvertedPendulum/runObserverControl.m

```diff
@@ -0,0 +1,45 @@
+function runObserverControl
+
+r = TWIP();
+[A,B] = r.linearize(0, zeros(4,1), 0);
+A = full(A); B = full(B);
+C = [0,0,1,0;1,-1,0,0]*180/pi;
+
+%observer design by pole placement
+omega0 = 25;
+L = place(A', C', -omega0*[1,2,3,4])';
+v = TWIPVisualizer(r);
+
+%construct linear state observer
+observer = LuenbergerObserver(r, L, []);
+observer.forward_model = LinearSystem(A, B, [], [], C, []);
+
+%construct LQR controller
+controller = r.balanceLQR();
+
+%wire up the system
+sysCl = buildClosedLoopObserverControl(r, observer, controller);
+
+xtraj = sysCl.simulate([0, 5]);
+v.playback(xtraj)
+
+function sysCl = buildClosedLoopObserverControl(plant, observer, controller)
```

Comment:
yikes.  agreed that we should put this functionality up in the observer class at some point, and that it's ok to do it locally first.  but i think it's important to maintain the abstract above simulink (so we don't depend on it, for instance). 

i think you could do the same thing with a call to mimoCascade, then mimoFeedback.  e.g. 
```
mimoFeedback(mimoCascade(plant,observer),controller)
```
i hope that the auto-wiring would work here (probably we should make it work if it doesn't)